src/auth-service/utils/mailer.js

```diff
@@ -518,7 +518,84 @@ const mailer = {
       };
     }
   },
+  updateForgottenPassword: async (email, firstName, lastName) => {
+    try {
+      const mailOptions = {
+        from: {
+          name: constants.EMAIL_NAME,
+          address: constants.EMAIL,
```

Comment:
the utils for the NEW email notifications (templates) when user either resets or updates their password